.github/ISSUE_TEMPLATE/1-bug_report.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug report 🐛
 description: Existing feature does not behave as expected.
-labels: [ "kind/bug", "status/to verify" ]
+labels: [ kind/bug, status/to verify ]
```

Comment:
i'm surprised this works :D 
how does it know it's 2 labels and one single label with comma char inside ?